stats/opentelemetry/e2e_test.go

```diff
@@ -94,23 +69,16 @@ func setup(t *testing.T, tafOn bool, maf func(string) bool) (*metric.ManualReade
 			}
 		},
 	}
-	var taf func(string) bool
-	if tafOn {
-		taf = func(str string) bool {
-			return str != ss.Target
-		}
-	}
+
 	if err := ss.Start([]grpc.ServerOption{ServerOption(Options{
 		MetricsOptions: MetricsOptions{
 			MeterProvider:         provider,
 			Metrics:               DefaultMetrics,
-			TargetAttributeFilter: taf,
 			MethodAttributeFilter: maf,
 		}})}, DialOption(Options{
 		MetricsOptions: MetricsOptions{
 			MeterProvider:         provider,
 			Metrics:               DefaultMetrics,
-			TargetAttributeFilter: taf,
 			MethodAttributeFilter: maf,
```

Comment:
Does the `MethodAttributeFilter` make sense on the client side? Why is it included in the dial option?

Reply:
Ah, good point. See https://github.com/grpc/grpc-go/blob/master/stats/opentelemetry/opentelemetry.go#L135 this comment. This only applies to server side. Deleted from here.